docs/source/FPGA-to-bitstream/VPR compilation.rst

```diff
@@ -1,7 +1,7 @@
 VPR compilation
 ===============
 
-Compile BLIF to FASM by VPR <-- architecture.xml + routing_resources.xml
+Compile BLIF to FASM by VPR ..|larr| architecture.xml + routing_resources.xml
```

Comment:
As above, maybe worth making this clearer - something like

```suggestion
VPR can compile a circuit in BLIF format to FASM (the FPGA Assembly format, describing a concrete list of features on a specific FPGA fabric to be enabled) using the architectural description in architecture.xml and routing_resources.xml
```

Reply:
Same as above, do we keep it with the new formulation or remove it? I think it's a good summary of what VPR does from a user perspective.

Reply:
I would say the paragraph directly below it is already a decent summary of the page, so the two paragraphs are just repeating each other at the moment. Could drop one or merge them, up to you!

Reply:
Yeah you're right, only the `BLIF` is missing. I will add it to the exisiting paragraphs.